app/client/jsx/Room.jsx

```diff
@@ -166,9 +169,11 @@ class Room extends Component {
 
     computeMapState(room) {
         const mapAlreadyUnlocked = LocalStorage.get("MAP_UNLOCKED")
-        const mapUnlockThreshold = 3
+        const mapUnlockThreshold = parseInt(Config.mapUnlockThreshold) || 3
```

Comment:
This `3` makes me think we should have a _generic_ config that sets defaults and is overridden by a user-specified config. Default values could be specified in another config file or programmatically, wherever we parse the user config